pyproject.toml

```diff
@@ -1,5 +1,10 @@
 [build-system]
-requires = ["setuptools>=67.8", "cython~=3.0.2", "numpy>=1.25,<2"]
+requires = [
+    "setuptools>=67.8",
+    "wheel",
+    "cython~=3.0.2",
+    "numpy>=2.0.0rc1"
```

Comment:
You have to use `==` with pre-release versions.
```suggestion
    "numpy==2.0.0rc1"
```

Reply:
@snowman2 in theory, pip treats `>=` like it does `==`, pre-releases are allowed: https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions. And it works in practice, too!

Reply:
Nice! I have had issues with that not working in the past and have avoided using that syntax since. Must have been years ago... I am glad to hear that it works now.